DFC.ServiceTaxonomy.GraphSync/GraphSyncers/Parts/PageLocationPartGraphSyncer.cs

```diff
@@ -0,0 +1,72 @@
+using System;
+using System.Threading.Tasks;
+using DFC.ServiceTaxonomy.GraphSync.GraphSyncers.Interfaces;
+using DFC.ServiceTaxonomy.PageLocation.Models;
+using Neo4j.Driver;
+using Newtonsoft.Json.Linq;
+
+namespace DFC.ServiceTaxonomy.GraphSync.GraphSyncers.Parts
+{
+#pragma warning disable S1481 // Unused local variables should be removed
+    public class PageLocationPartGraphSyncer : IContentPartGraphSyncer
+    {
+        public string PartName => nameof(PageLocationPart);
+
+        private static readonly Func<string, string> _pageLocationPropertyNameTransform = n => $"pagelocation_{n}";
+
+        private const string
+            UrlNamePropertyName = "UrlName",
+            DefaultPageForLocationPropertyName = "DefaultPageForLocation",
+            FullUrlPropertyName = "FullUrl";
+
+        public async Task AddSyncComponents(JObject content, IGraphMergeContext context)
+        {
+            using var _ = context.GraphSyncHelper.PushPropertyNameTransform(_pageLocationPropertyNameTransform);
+
+            JValue? value = (JValue?)content[UrlNamePropertyName];
```

Comment:
There's helper methods for this now as extensions on `IMergeNodeCommand`, so you can e.g. do this...
```
context.MergeNodeCommand.AddProperty<string>()
```